libvmaf/src/feature/integer_vif.c

```diff
@@ -618,6 +620,11 @@ static int init(VmafFeatureExtractor *fex, enum VmafPixelFormat pix_fmt,
     }
 #endif
 */
+#elif ARCH_AARCH64
+    s->subsample_rd_8 = vif_subsample_rd_8_neon;
```

Comment:
For the x86 optimizations, it is possible to mask specific instruction sets at runtime by setting `VmafConfiguration.cpu_mask`. Is there something similar that makes sense for NEON? At the very least, we should be able to fallback to the C path at runtime.

Reply:
Hi @kylophone 
It is possible to mask some NEON architectures, yet, I am using only the basic intrinsic in my implementation, which are supported by every arm64 platform, so there is actually nothing I can mask.

Reply:
Thanks. I would say let's not worry about it at this point. I will probably end up porting some similar masking logic at some point.